deps.windows/60-python-arm64.ps1

```diff
@@ -0,0 +1,65 @@
+param(
```

Comment:
Use PyEnv for this - arm64 support is in an alpha state (iirc) and PyEnv support is forthcoming. Until this version of Python is stable enough to ship with OBS, PyEnv support should be available.

Until then, skip building Python for obs-deps.

Reply:
disabled